src/backend/api/mirror_processor.py

```diff
@@ -89,14 +89,14 @@ def __init__(self, logger: Logger):
             force_close=True,
         )
         self.retry_options = ExponentialRetry(
-            attempts=3,
+            attempts=2,
```

Comment:
Waiting on the retries is just a blocker for the update script.  Any mirror that's only intermittently available and presumably unstable doesn't belong in the mirrorlist anyway for the best UX.

With the new updates running every 15 minutes mirrors can be added back much more quickly than previously.

Reply:
I guess we will lose unknown amount of mirrors with more often update, because they can block us.

Reply:
> I guess we will lose unknown amount of mirrors with more often update, because they can block us.

That's a whole nother can of worms.  Mirrors shouldn't have such strict firewalls in place because that will create bad UX.  If our few requests checking a mirror's status gets blocked think about how many users will hit weird firewall rules on that mirror as well.